package.json

```diff
@@ -8,11 +8,12 @@
   "scripts": {
     "build:load-schema": "node --loader @esbuild-kit/esm-loader scripts/schema-loader.ts",
```

Comment:
Yes, that's expected behavior. If you try to override a schema, it downloads and re-generates a schema for `web-types`. However, it stays the same because there are no changes in the schema and generator. This option is made just in case the schema or schema transformer is changed and we need to generate and commit a new schema.